Packs/EmailCommunication/Scripts/PreprocessEmail/PreprocessEmail.py

```diff
@@ -108,36 +162,68 @@ def get_attachments_using_instance(email_related_incident, labels):
         demisto.debug('Attachments could only be retrieved from EWS v2 or Gmail')
 
 
+def get_incident_related_files(incident_id):
+    """Get the email reply attachments after they were uploaded to the server and saved
+    to context of the email reply related incident.
+
+    Args:
+        incident_id (str): The ID of the incident whose context we'd like to get.
```

Comment:
why `incident_id` here is a string and in the `update_latest_message_field` it is a number